main.tex

```diff
@@ -92,75 +87,69 @@ \section*{Introduction}
 
 %% Paragraph 1: The relevance of insecticide resistance to malaria. Start broad, explain why insecticide resistance is relevant to malaria control. Briefly review some of the evidence of the actual and potential impact of insecticide resistance on intervention efficacy and disease prevalence.
 %
-An estimated 663 million cases of malaria were averted in Africa between 2000 and 2015 due to public health interventions, of which 68\% were prevented by insecticide-treated bed-nets (ITNs) and @@N\% through indoor residual spraying of insecticides (IRS).
+An estimated 663 million cases of malaria were averted in Africa between 2000 and 2015 due to public health interventions, of which 68\% were prevented by insecticide-treated bed-nets (ITNs) and 10\% through indoor residual spraying of insecticides (IRS) \cite{Bhatt2015}.
 %
-However, over this same period, insecticide resistance has become increasingly prevalent in malaria vector populations.
+However, over this same period, insecticide resistance has become increasingly prevalent in malaria vector populations \cite{Hemingway2016}, \cite{Silva2014}.
 %
-Four chemical classes of insecticides -- organophosphates, carbamates, pyrethroids and organochlorines -- are licensed for use in public health, but only pyrethroids are approved by the World Health Organisation (WHO) for use in ITNs.
+Four chemical classes of insecticides – organophosphates, carbamates, pyrethroids and organochlorines – are licensed for use in public health, but only pyrethroids are approved by the World Health Organisation (WHO) for use in ITNs.
```

Comment:
I think these should actually be [em dashes](https://tex.stackexchange.com/questions/3819/dashes-vs-vs), i.e.:

```
Four chemical classes of insecticides---organophosphates, carbamates, pyrethroids and organochlorines---are licensed ...
```